packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -138,7 +138,27 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
         STARTING_TIMESTAMP = _startingTimestamp;
         L2_BLOCK_TIME = _l2BlockTime;
 
-        initialize(_genesisL2Output, _startingBlockNumber, _proposer, _owner);
+        initialize(_genesisL2Output, _proposer, _owner);
+    }
+
+    /**
+     * @notice Initializer.
+     *
+     * @param _genesisL2Output     The initial L2 output of the L2 chain.
+     * @param _proposer            The address of the proposer.
+     * @param _owner               The address of the owner.
+     */
+    function initialize(
```

Comment:
Moving this up to align with other contracts where initialize is underneath the constructor